### ISSUE TITLE
App Services: Early Access: Tracking status of Application Insights?

### ISSUE THREAD

**Issue by JeremyCaney** [Azure/app-service-linux-docs#61]
On the [.NET on App Services](https://github.com/Azure/app-service-linux-docs/blob/master/Runtime_Support/dot_net_core.md#net-5-early-access-limitations) page, under [.NET 5 Early Access limitations
](https://github.com/Azure/app-service-linux-docs/blob/master/Runtime_Support/dot_net_core.md#net-5-early-access-limitations), it notes:

> [Application Insights](https://azure.microsoft.com/services/monitor) is currently not supported for .NET 5 apps, we are working with the Application Insights team to resolve this.

My team is reluctant to update a number of App Services to ASP.NET Core 5 until this issue is resolved, since they don't include explicit implementation of Application Insights in the code. Given that, I'd like to monitor the progress of the Application Insights support for the App Services Early Access feature. 

Is there an issue we can follow to stay informed of the status? If not, can we use this issue as a way of updating the community?

**Comment by naeem76** [comment]
Changed an app service to use net 5, application insights was greyed out on azure portal, but even after changing back to net core, and publishing a net core 3.1 app, app insights is still greyed out

**Comment by JeremyCaney** [comment]
@naeem76: That's consistent with my experience. If you change it back to .NET Core, a prompt _should_ appear at the top of the App Services _Overview_ pane:

> Click here to access Application Insights for monitoring and profiling for your ASP.NET Core app.

I'm not sure if that will (allow you to) reenable Application Insights for a .NET 3.x app, though. It certainly doesn't with an ASP.NET Core 5 app—even if it's setup to use a _self contained_ deployment mode. 

All around, it seems like there are some goofy hacks to mitigate the current lack of Application Insights support, and they lead to a confusing user experience with minimal communication to customers. 

I assume that's just because these were put in at the last minute due to Application Insights not being ready by launch. But this would definitely benefit from, at minimum, improved messaging and rollback abilities if Application Insights is going to be a while.

**Comment by naeem76** [comment]
Yes the prompt is coming now, and I can also go directly to the Insights resource directly and its working. 

I'm still waiting on official support like you now, and yes a issue tracker would be good to have, I ported from net 2.1 to net 5 and then back to net core3.1 just because of Application Insights

**Comment by IanKemp** [comment]
> Application Insights not being ready by launch

It absolutely was: https://github.com/microsoft/ApplicationInsights-dotnet/issues/1960#issuecomment-725256663

It seems that the issue is entirely with Azure.
